modules/ppcp-wc-gateway/src/Gateway/CreditCardGateway.php

```diff
@@ -466,7 +466,7 @@ public function process_payment( $order_id ) {
 						continue;
 					}
 
-					$custom_id    = $wc_order->get_order_number();
+					$custom_id    = $wc_order->get_id();
```

Comment:
```suggestion
					$custom_id    = (string) $wc_order->get_id();
```
It should be cast to a string, because a sting is expected from` create_order()`.